core/miner/storage_fsm/impl/checks.cpp

```diff
@@ -256,6 +278,8 @@ OUTCOME_CPP_DEFINE_CATEGORY(fc::mining::checks, ChecksError, e) {
       return "ChecksError: invalid proof";
     case E::kCommitWaitFail:
       return "ChecksError: need to wait commit";
+    case E::kMinerVersion:
+      return "ChecksError::kMinerVersion";
```

Comment:
Please describe more clear

Reply:
Searching error code in code is easier to understand where the error happened and what it means.